controllers/compassmanager_controller.go

```diff
@@ -151,8 +154,8 @@ func (cm *CompassManagerReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 
 	// Kubeconfig doesn't exist / is empty
 	if isNotFound(err) || len(kubeconfig) == 0 {
-		cm.Log.Infof("Kubeconfig for Kyma resource %s not available.", req.Name)
-		return ctrl.Result{RequeueAfter: cm.requeueTime}, nil
+		cm.Log.Infof("Kubeconfig for Kyma resource %s not available. Next attempt in 3 minutes", req.Name)
```

Comment:
`Next attempt in 3 minutes` will be false if the `requeueTimeForKubeconfig` is changed to a different value. 
Could it be changed e.g. to use the current `requeueTimeForKubeconfig`?